doc/changes/10923.md

```diff
@@ -0,0 +1,3 @@
+- Fix error/warning messages displayed when the specified dune configuration
```

Comment:
I would say what it fixes is the file that the error refers to, maybe worth being explicit about it in the changelog.